keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -287,15 +297,10 @@ def compile(
                 A preconfigured `FocalLoss` is provided when the string "focal" is
                 passed.
             weight_decay: a float for variable weight decay.
+            metrics: object detection metrics.
```

Comment:
This warrants some more docstring content -- specifically, that it will break TPU compatibility and make decoding happen during training

Reply:
Done and updated